packages/app/web/index.html

```diff
@@ -45,6 +45,7 @@
 
       body {
         display: flex;
+        flex-direction: column;
```

Comment:
I think this is the right thing to do! The alternative would often produce strange overlaps.